kitchen-tests/cookbooks/end_to_end/recipes/default.rb

```diff
@@ -36,7 +36,9 @@
   only_if { platform_family?("rhel") }
 end
 
-build_essential
+build_essential do
+  raise_if_unsupported true
```

Comment:
We shouldn't be using this in the kitchen tests. It would be better to have it blow up in our faces when we add new platforms so we can make sure we add support for those.

Reply:
Right but that's what it does now by raising?

Reply:
Slowly walking away from keyboard. Yep I'm an idiot